traefik/datadog_checks/traefik/traefik.py

```diff
@@ -42,6 +42,11 @@ def check(self, instance):
                 else:
                     self.log.warning('Field total_count not found in response.')
 
+                if 'average_response_time_sec' in payload:
+                    self.gauge('traefik.average_response_time_sec', payload['average_response_time_sec'])
+                else:
+                    self.log.warning('Field average_response_time_sec not found in response.')
```

Comment:
```suggestion
                    self.log.debug('Field average_response_time_sec not found in response.')
```

Can we lower these to debug level since warning can be a noisy

Reply:
Yup, have converted all of the `warning` log levels to `debug`.